api/src/controllers/signin.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const router = express.Router();
+const Joi = require("joi");
+const jwt = require("jsonwebtoken");
+const { getToken } = require("../passport");
+const config = require("../config");
+const { serializeYoung, serializeReferent } = require("../utils/serializer");
+const { ERRORS } = require("../utils");
+
+const Young = require("../models/young");
+const Referent = require("../models/referent");
+const { capture } = require("../sentry");
+
+router.get("/signin_token", async (req, res) => {
+  try {
+    const token = getToken(req);
+    if (!token) return res.status(200).send({ ok: false, user: { restriction: "public" } });
+    const jwtPayload = await new Promise((resolve, reject) => {
+      jwt.verify(token, config.secret, function (err, decoded) {
+        if (err) reject(err);
+        resolve(decoded);
+      });
+    });
+    if (!jwtPayload) return res.status(200).send({ ok: false, user: { restriction: "public" } });
```

Comment:
Si j'ai bien compris (et pour être cohérent avec la ligne 26) `ok` doit être `true` ici.

```suggestion
    if (!jwtPayload) return res.status(200).send({ ok: true, user: { restriction: "public" } });
```